ui/src/translations/en.json

```diff
@@ -91,6 +91,7 @@
   "volume_creation_success": "Volume {name} is being created.",
   "volume_creation_failed": "Volume {name} creation has failed.",
   "name_error": "Name must be lowercase letters, numbers and hyphens.",
+  "label_error": "Label must be letters, numbers or hyphens and must begin and end with a letter or a number.",
```

Comment:
That's not really accurate, description from K8s:
```
A valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```

Reply:
Should I change it to: "Label must be letters, numbers, hyphens, dots or underscores and must begin and end with a letter or a number." ?
Or like the example: "A valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345')"

Reply:
As you prefer, both are correct.